thorlcr/msort/tsortm.cpp

```diff
@@ -688,7 +642,6 @@ class CSortMaster : public IThorSorterMaster, public CSimpleInterface
             }
         } afor3(slaves, splitMap, numnodes, numsplits);
         afor3.For(numnodes, 20, true);
-#ifdef _TRACE
 #ifdef TRACE_PARTITION
```

Comment:
if (!REJECTLOG(MCthorDetailedDebugInfo)) before the #ifdef here ?

Reply:
I didn't change every existing compile time directive that controlled debug tracing like this.
There is a bit of overhead even with REJECTLOG.
Could change, but don't think it's warranted for this and most other compile time section of tracnig.